Lib/shutil.py

```diff
@@ -665,6 +665,8 @@ def onexc(*args):
     if _use_fd_functions:
         return _rmtree_safe_fd(dir_fd, path, onexc)
     else:
+        if dir_fd is not None:
```

Comment:
Should it be

```suggestion
        if dir_fd is None:
```

?

Reply:
The situation arises if a user on a system that doesn't support `dir_fd` (e.g. Windows) calls `shutil.rmtree('blah', dir_fd=somevalue)`, where *somevalue* isn't none. I think the condition is correct!

Reply:
My apologises, I have read the chunk out of context (skipping the code above to save time). As a result, I misinterpreted the intention of the chunk as *if `dir_fd` is not available, we cannot continue so report and abort*.